.github/workflows/get-merge-commit.yml

```diff
@@ -32,6 +40,7 @@ jobs:
           sparse-checkout: ci
 
       - name: Check if the PR can be merged and get the test merge commit
+        if: inputs.leader == true
```

Comment:
I know these conditions look stupid, but I genuinely could not get it to work otherwise. lol.